src/meshes/p4est_mesh.jl

```diff
@@ -1492,12 +1765,12 @@ end
 # routines found in `mappings_geometry_curved_2d.jl` or `mappings_geometry_straight_2d.jl`
 function calc_tree_node_coordinates!(node_coordinates::AbstractArray{<:Any, 4},
                                      file_lines::Vector{String}, nodes, vertices, RealT)
-    # Get the number of trees and the number of interpolation nodes
+    # Get the number of trees (elements) and the number of interpolation nodes
     n_trees = last(size(node_coordinates))
     nnodes = length(nodes)
 
     # Setup the starting file index to read in element indices and the additional
-    # curved boundary information provided by HOHQMesh.
+    # higher-order, polygonial boundary information provided by HOHQMesh.
```

Comment:
Again, I would prefer to call the boundaries curved rather than polygonal. I already mentioned two reasons above. Another is that it over complicates the nomenclature. I think you want to use "polygonal" to clarify that the boundary is a closed, simply connected region with some number of sides (that are possibly individually curved). That is, some segments of the polygon can be straight-sided, attached to a spline and then attached to circular arc. This is why I think the catch all term of "curved" is better because it encompassed all these cases (as well as dimensions). 

After all, one could say that a simple straight sided box domain has "polygonal boundary information" for the same reasons, but people would think I am weird to refer to a Carteisan box in this way :)